In [1]:
import torch,os
device=torch.device("cuda:6")

In [2]:
from ultralytics import YOLO
import torch
import cv2
import time
from PIL import Image
import yaml

In [3]:
#JUST DOCS MODEL TRAINING

yaml_filename = 'sample.yaml'


model = YOLO('yolov8n.yaml') # creates Yolo object from 'yolov8n.yaml' configuration file. 
model = YOLO('yolov8n.pt')   # Loads pretrained weights             
model = YOLO('yolov8n.yaml').load('yolov8n.pt')  # build from YAML and transfer weights

result_just_docs=model.train(data='{}'.format(yaml_filename), epochs=500,patience=30, batch=32,  imgsz=640, device=3)

Transferred 355/355 items from pretrained weights
New https://pypi.org/project/ultralytics/8.3.99 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.49 🚀 Python-3.9.16 torch-2.0.1+cu117 CUDA:3 (NVIDIA RTX A5000, 24123MiB)
engine/trainer: task=detect, mode=train, model=yolov8n.yaml, data=sample.yaml, epochs=500, time=None, patience=30, batch=32, imgsz=640, save=True, save_period=-1, cache=False, device=3, workers=8, project=None, name=train15, exist_ok=False, pretrained=yolov8n.pt, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_

train: Scanning /mnt/hdd1/gauravsharma/Rishabh/dent_scratch_yolo/soldiers_dataset/labels/train... 270 images, 0 backgrounds, 0 corrupt: 100%|██████████| 270/270 [00:00<00:00, 1027.37it/s]

train: New cache created: /mnt/hdd1/gauravsharma/Rishabh/dent_scratch_yolo/soldiers_dataset/labels/train.cache


albumentations: Blur(p=0.01, blur_limit=(3, 7)), MedianBlur(p=0.01, blur_limit=(3, 7)), ToGray(p=0.01), CLAHE(p=0.01, clip_limit=(1, 4.0), tile_grid_size=(8, 8))


val: Scanning /mnt/hdd1/gauravsharma/Rishabh/dent_scratch_yolo/soldiers_dataset/labels/val.cache... 30 images, 0 backgrounds, 0 corrupt: 100%|██████████| 30/30 [00:00<?, ?it/s]


Plotting labels to runs/detect/train15/labels.jpg... 
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.001429, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0)
Image sizes 640 train, 640 val
Using 8 dataloader workers
Logging results to runs/detect/train15
Starting training for 500 epochs...

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      1/500      4.49G      1.932      3.165      1.845        133        640: 100%|██████████| 9/9 [00:04<00:00,  1.91it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  1.64it/s]

                   all         30        135     0.0173      0.956      0.218      0.112



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      2/500      4.45G      1.455      2.603      1.417        117        640: 100%|██████████| 9/9 [00:01<00:00,  5.47it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.42it/s]

                   all         30        135     0.0145      0.956      0.317      0.187



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      3/500      4.52G      1.452      2.065      1.391        136        640: 100%|██████████| 9/9 [00:01<00:00,  5.48it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.67it/s]

                   all         30        135      0.442      0.301      0.338      0.183



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      4/500      4.51G       1.44      1.874      1.404        121        640: 100%|██████████| 9/9 [00:01<00:00,  5.63it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.89it/s]

                   all         30        135      0.536      0.311      0.344      0.155



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      5/500      4.58G      1.405      1.757      1.397        116        640: 100%|██████████| 9/9 [00:01<00:00,  5.60it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.65it/s]

                   all         30        135      0.406      0.286      0.272      0.143



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      6/500      4.54G      1.414      1.688      1.378        112        640: 100%|██████████| 9/9 [00:01<00:00,  5.96it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.56it/s]

                   all         30        135      0.548      0.498      0.521      0.278



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      7/500      4.56G      1.333      1.473       1.33        121        640: 100%|██████████| 9/9 [00:01<00:00,  6.68it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.75it/s]

                   all         30        135       0.47      0.337      0.375      0.173



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      8/500      4.51G      1.352       1.44      1.333        176        640: 100%|██████████| 9/9 [00:01<00:00,  6.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.71it/s]

                   all         30        135      0.687      0.338      0.404       0.23



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


      9/500      4.52G      1.323      1.382      1.308        166        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.57it/s]

                   all         30        135      0.527      0.428       0.47      0.231



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     10/500      4.52G      1.372      1.397      1.357        124        640: 100%|██████████| 9/9 [00:01<00:00,  6.29it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.42it/s]

                   all         30        135      0.414      0.517      0.435       0.21



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     11/500      4.52G       1.33      1.359      1.314        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.59it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.74it/s]

                   all         30        135      0.561      0.474      0.486      0.232



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     12/500      4.48G      1.363      1.326      1.335        101        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.36it/s]

                   all         30        135      0.441      0.412      0.262      0.106



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     13/500      4.57G      1.372      1.312      1.337        118        640: 100%|██████████| 9/9 [00:01<00:00,  6.41it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.48it/s]

                   all         30        135      0.377      0.529      0.424        0.2



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     14/500      4.51G      1.336      1.291      1.311        120        640: 100%|██████████| 9/9 [00:01<00:00,  6.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.53it/s]

                   all         30        135      0.477      0.454      0.468       0.23



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     15/500      4.44G      1.307      1.244      1.307        114        640: 100%|██████████| 9/9 [00:01<00:00,  6.55it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.46it/s]

                   all         30        135      0.427        0.5      0.495       0.25



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     16/500       4.5G      1.332      1.225      1.315        109        640: 100%|██████████| 9/9 [00:01<00:00,  6.29it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.54it/s]

                   all         30        135      0.268      0.518      0.363      0.145



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     17/500      4.53G      1.295      1.167      1.299        119        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.60it/s]

                   all         30        135      0.679      0.338      0.506      0.255



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     18/500      4.49G      1.258      1.176      1.266        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.30it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.59it/s]

                   all         30        135      0.462      0.519      0.479       0.24



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     19/500      4.49G      1.292      1.144      1.316        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.50it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.76it/s]

                   all         30        135      0.534      0.571      0.576       0.27



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     20/500      4.48G      1.256      1.106      1.277        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.27it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.66it/s]

                   all         30        135       0.48      0.485      0.417      0.225



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     21/500      4.51G      1.252      1.095      1.264        113        640: 100%|██████████| 9/9 [00:01<00:00,  6.59it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.70it/s]

                   all         30        135      0.306      0.358      0.277      0.125



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     22/500      4.53G      1.228      1.068      1.249        121        640: 100%|██████████| 9/9 [00:01<00:00,  6.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.65it/s]

                   all         30        135      0.479      0.488      0.483      0.227



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     23/500      4.46G      1.252      1.087      1.267        135        640: 100%|██████████| 9/9 [00:01<00:00,  6.61it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.73it/s]

                   all         30        135      0.382      0.355      0.347      0.149



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     24/500      4.51G      1.267      1.075      1.259        130        640: 100%|██████████| 9/9 [00:01<00:00,  6.05it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.73it/s]

                   all         30        135      0.425       0.59       0.48      0.254



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     25/500      4.49G      1.198      0.999      1.222        104        640: 100%|██████████| 9/9 [00:01<00:00,  5.55it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  3.81it/s]

                   all         30        135      0.641      0.537      0.596       0.32



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     26/500      4.48G      1.222      1.017      1.244        109        640: 100%|██████████| 9/9 [00:01<00:00,  5.74it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.74it/s]

                   all         30        135      0.679      0.634       0.69      0.371



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     27/500       4.5G      1.206      1.017      1.232        114        640: 100%|██████████| 9/9 [00:01<00:00,  6.22it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.64it/s]

                   all         30        135      0.756      0.699      0.773      0.428



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     28/500      4.51G      1.176     0.9311      1.211        119        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.69it/s]

                   all         30        135       0.63      0.725      0.744      0.373



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     29/500      4.52G      1.188     0.9605      1.214        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.49it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.76it/s]

                   all         30        135       0.61      0.394      0.414      0.185



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     30/500       4.5G      1.167     0.9319      1.203        170        640: 100%|██████████| 9/9 [00:01<00:00,  6.33it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.63it/s]

                   all         30        135      0.707      0.637      0.673      0.375



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     31/500      4.53G      1.171     0.9642      1.217        115        640: 100%|██████████| 9/9 [00:01<00:00,  6.52it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.74it/s]

                   all         30        135       0.62      0.455      0.528      0.275



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     32/500      4.48G      1.169     0.9585      1.216        117        640: 100%|██████████| 9/9 [00:01<00:00,  6.30it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.74it/s]

                   all         30        135      0.333      0.436      0.379       0.21



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     33/500       4.5G      1.146     0.9477      1.212        133        640: 100%|██████████| 9/9 [00:01<00:00,  6.55it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.83it/s]

                   all         30        135      0.423      0.403      0.358      0.219



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     34/500      4.52G      1.134     0.9477      1.199        112        640: 100%|██████████| 9/9 [00:01<00:00,  6.28it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.78it/s]

                   all         30        135      0.702      0.494      0.514      0.306



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     35/500      4.51G      1.143     0.9418      1.182        121        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.75it/s]

                   all         30        135      0.565      0.593      0.604      0.345



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     36/500      4.47G       1.11     0.9184      1.195         93        640: 100%|██████████| 9/9 [00:01<00:00,  6.21it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.59it/s]

                   all         30        135      0.684      0.521      0.624      0.319



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     37/500      4.52G      1.102      0.858      1.183        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.79it/s]

                   all         30        135      0.598      0.593      0.588       0.34



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     38/500      4.51G      1.136     0.8878      1.196        123        640: 100%|██████████| 9/9 [00:01<00:00,  6.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.75it/s]

                   all         30        135        0.4      0.423      0.367      0.175



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     39/500      4.53G      1.093     0.8896      1.186        130        640: 100%|██████████| 9/9 [00:01<00:00,  6.50it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.617      0.494      0.506      0.267



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     40/500      4.51G      1.124     0.8907      1.197        139        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.80it/s]

                   all         30        135      0.739      0.675      0.684      0.413



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     41/500       4.5G      1.121     0.8475      1.185        130        640: 100%|██████████| 9/9 [00:01<00:00,  6.48it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.83it/s]

                   all         30        135      0.812       0.79       0.82      0.483



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     42/500      4.61G      1.089     0.8404      1.171        132        640: 100%|██████████| 9/9 [00:01<00:00,  6.28it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.80it/s]

                   all         30        135      0.849      0.733      0.874      0.553



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     43/500       4.5G       1.06     0.8678      1.174        117        640: 100%|██████████| 9/9 [00:01<00:00,  5.94it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.01it/s]

                   all         30        135      0.662      0.715      0.687      0.412



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     44/500      4.51G      1.063     0.8197      1.159        123        640: 100%|██████████| 9/9 [00:01<00:00,  4.84it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.835       0.68      0.802      0.477



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     45/500      4.51G      1.084     0.8273       1.16        136        640: 100%|██████████| 9/9 [00:01<00:00,  6.55it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.832      0.685      0.795      0.488



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     46/500      4.49G      1.046     0.8061       1.15        114        640: 100%|██████████| 9/9 [00:01<00:00,  6.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.735      0.562      0.655      0.406



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     47/500       4.5G      1.062     0.8016      1.143        111        640: 100%|██████████| 9/9 [00:01<00:00,  6.42it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135        0.6      0.447      0.479      0.263



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     48/500      4.51G      1.058      0.786      1.138        130        640: 100%|██████████| 9/9 [00:01<00:00,  6.33it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.789      0.609      0.691      0.405



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     49/500      4.58G      1.025     0.7685       1.13        148        640: 100%|██████████| 9/9 [00:01<00:00,  6.48it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.86it/s]

                   all         30        135      0.832      0.732      0.825      0.492



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     50/500      4.49G     0.9971     0.7706      1.117        140        640: 100%|██████████| 9/9 [00:01<00:00,  6.32it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.851      0.676      0.745      0.432



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     51/500      4.52G      1.025     0.7531       1.13        146        640: 100%|██████████| 9/9 [00:01<00:00,  6.54it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135       0.69      0.836      0.868      0.527



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     52/500      4.51G      1.012     0.7321       1.12        106        640: 100%|██████████| 9/9 [00:01<00:00,  6.39it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.648       0.71      0.727      0.418



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     53/500      4.52G      1.004       0.75      1.116        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.729      0.706      0.783      0.468



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     54/500      4.48G      1.038     0.7392      1.148        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.36it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  3.86it/s]

                   all         30        135      0.786      0.671      0.779       0.46



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     55/500      4.54G      1.013     0.7708      1.136        112        640: 100%|██████████| 9/9 [00:01<00:00,  4.67it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  2.59it/s]

                   all         30        135      0.715      0.821      0.847      0.505



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     56/500      4.47G     0.9784     0.7313      1.112        137        640: 100%|██████████| 9/9 [00:01<00:00,  6.20it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.91it/s]

                   all         30        135      0.871      0.672       0.82      0.505



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     57/500      4.49G      1.017     0.7415      1.122        144        640: 100%|██████████| 9/9 [00:01<00:00,  6.56it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.89it/s]

                   all         30        135      0.854      0.791      0.882      0.582



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     58/500      4.47G      1.009     0.7235      1.119        105        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.742      0.869      0.867      0.573



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     59/500      4.49G      1.024     0.7553      1.127        140        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.87it/s]

                   all         30        135       0.85      0.857      0.911      0.601



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     60/500      4.51G     0.9834      0.731      1.122        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.42it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.89it/s]

                   all         30        135      0.863      0.802      0.878      0.574



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     61/500      4.49G     0.9899     0.7276      1.118        132        640: 100%|██████████| 9/9 [00:01<00:00,  6.52it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.86it/s]

                   all         30        135      0.813      0.828      0.873      0.564



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     62/500       4.5G     0.9661     0.7091      1.101        121        640: 100%|██████████| 9/9 [00:01<00:00,  5.06it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  3.86it/s]

                   all         30        135      0.837      0.783      0.881      0.555



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     63/500      4.56G     0.9453     0.6851      1.094        130        640: 100%|██████████| 9/9 [00:01<00:00,  4.74it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  2.98it/s]

                   all         30        135      0.903      0.849      0.938      0.643



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     64/500      4.48G     0.9169     0.6545      1.081        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.91it/s]

                   all         30        135      0.855      0.782      0.895      0.574



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     65/500       4.5G     0.9752     0.7111       1.11        123        640: 100%|██████████| 9/9 [00:01<00:00,  6.56it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.88it/s]

                   all         30        135      0.835      0.801      0.887      0.562



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     66/500      4.53G     0.9699     0.6873      1.111        134        640: 100%|██████████| 9/9 [00:01<00:00,  6.33it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.73it/s]

                   all         30        135      0.896       0.86       0.93      0.589



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     67/500       4.5G     0.9295     0.6675      1.085        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.44it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.88it/s]

                   all         30        135      0.888      0.897      0.947      0.642



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     68/500      4.48G     0.9221      0.688      1.089        128        640: 100%|██████████| 9/9 [00:01<00:00,  6.41it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.776      0.845      0.907      0.604



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     69/500      4.55G     0.9239     0.6694      1.088        142        640: 100%|██████████| 9/9 [00:01<00:00,  6.47it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.89it/s]

                   all         30        135      0.922      0.775       0.93       0.62



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     70/500      4.51G     0.9436     0.6724       1.11         88        640: 100%|██████████| 9/9 [00:01<00:00,  6.29it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.67it/s]

                   all         30        135      0.829      0.801      0.893      0.595



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     71/500      4.44G     0.9142     0.6605      1.089        100        640: 100%|██████████| 9/9 [00:01<00:00,  6.43it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.785      0.775      0.861      0.554



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     72/500      4.49G     0.9354     0.6633      1.094        124        640: 100%|██████████| 9/9 [00:01<00:00,  6.29it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.872      0.776      0.874      0.597



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     73/500      4.53G     0.8935     0.6482      1.073        107        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.783      0.848      0.911      0.643



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     74/500      4.49G     0.9232     0.6727      1.077        131        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.85it/s]

                   all         30        135      0.869      0.824      0.899      0.637



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     75/500      4.44G     0.9091     0.6503      1.086        110        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.92it/s]

                   all         30        135      0.906      0.771      0.898      0.594



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     76/500      4.49G     0.9019     0.6439      1.087        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.38it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.01it/s]

                   all         30        135      0.808      0.784      0.854      0.531



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     77/500      4.54G     0.9139     0.6508      1.091        138        640: 100%|██████████| 9/9 [00:01<00:00,  6.59it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135       0.91      0.874      0.929       0.64



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     78/500      4.49G      0.892     0.6227      1.074        128        640: 100%|██████████| 9/9 [00:01<00:00,  6.39it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135      0.796      0.768       0.84      0.561



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     79/500      4.53G      0.898     0.6424      1.085        112        640: 100%|██████████| 9/9 [00:01<00:00,  6.52it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.84it/s]

                   all         30        135      0.742      0.818      0.862      0.566



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     80/500       4.5G     0.8736     0.6187      1.049        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.38it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.99it/s]

                   all         30        135      0.834       0.89       0.94       0.61



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     81/500      4.44G     0.8854     0.6323      1.071        117        640: 100%|██████████| 9/9 [00:01<00:00,  6.58it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.843      0.905      0.933      0.646



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     82/500      4.47G     0.8749     0.6365      1.083        144        640: 100%|██████████| 9/9 [00:01<00:00,  6.37it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.867      0.873      0.937      0.667



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     83/500      4.52G     0.8706     0.6254      1.064        100        640: 100%|██████████| 9/9 [00:01<00:00,  6.64it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.678      0.844      0.837      0.579



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     84/500      4.52G     0.8682     0.6203      1.064         96        640: 100%|██████████| 9/9 [00:01<00:00,  6.37it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.86it/s]

                   all         30        135      0.833      0.881       0.91      0.635



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     85/500      4.52G     0.8807     0.6216      1.065        149        640: 100%|██████████| 9/9 [00:01<00:00,  6.58it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.92it/s]

                   all         30        135      0.804      0.781      0.852      0.586



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     86/500      4.59G     0.8549     0.6123      1.059         98        640: 100%|██████████| 9/9 [00:01<00:00,  6.37it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.07it/s]


                   all         30        135      0.857      0.685      0.786      0.489

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     87/500      4.52G     0.8487     0.6105      1.053        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.56it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.05it/s]


                   all         30        135      0.821      0.871      0.909       0.65

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     88/500      4.52G     0.8397     0.6062      1.054        111        640: 100%|██████████| 9/9 [00:01<00:00,  6.40it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.02it/s]

                   all         30        135      0.878      0.745      0.869      0.615



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     89/500      4.55G     0.8593     0.6178      1.058        147        640: 100%|██████████| 9/9 [00:01<00:00,  6.61it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135       0.89      0.824      0.912      0.644



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     90/500      4.48G     0.8616     0.6088      1.065        121        640: 100%|██████████| 9/9 [00:01<00:00,  6.36it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.80it/s]

                   all         30        135      0.854      0.855      0.936      0.676



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     91/500      4.56G     0.8565     0.6169      1.059        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.59it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.806      0.885       0.92      0.663



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     92/500      4.47G     0.8524      0.602      1.063        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.40it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.78it/s]

                   all         30        135      0.901      0.721      0.877      0.613



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     93/500      4.51G     0.8486     0.6194      1.058        118        640: 100%|██████████| 9/9 [00:01<00:00,  6.54it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.841      0.818       0.89      0.625



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     94/500      4.51G     0.8145     0.5832      1.044        133        640: 100%|██████████| 9/9 [00:01<00:00,  6.30it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  2.69it/s]

                   all         30        135      0.909       0.78       0.92      0.624



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     95/500       4.5G     0.8266     0.6094      1.035        116        640: 100%|██████████| 9/9 [00:02<00:00,  3.67it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  2.37it/s]

                   all         30        135      0.837      0.843      0.897      0.656



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     96/500      4.51G     0.8335     0.5952      1.048        101        640: 100%|██████████| 9/9 [00:01<00:00,  6.19it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135      0.825      0.862      0.909      0.639



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     97/500      4.44G     0.8112     0.5901      1.035        145        640: 100%|██████████| 9/9 [00:01<00:00,  6.54it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135      0.867      0.857      0.918      0.673



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     98/500      4.54G     0.8108     0.5719      1.027        132        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135        0.8      0.896      0.922      0.636



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


     99/500      4.52G     0.8101     0.5546       1.03        100        640: 100%|██████████| 9/9 [00:01<00:00,  6.58it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.901      0.865      0.941      0.656



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    100/500      4.45G     0.8093     0.5615      1.029        109        640: 100%|██████████| 9/9 [00:01<00:00,  6.38it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135       0.88      0.842      0.917      0.647



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    101/500      4.44G     0.8183     0.5615      1.038        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.58it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.01it/s]

                   all         30        135       0.83      0.813      0.882      0.633



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    102/500      4.51G     0.8219     0.5684      1.042        120        640: 100%|██████████| 9/9 [00:01<00:00,  6.39it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.00it/s]

                   all         30        135      0.896      0.902      0.943      0.679



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    103/500      4.49G     0.8146     0.5543      1.034        118        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.85it/s]

                   all         30        135       0.91      0.891      0.951      0.697



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    104/500      4.51G     0.7781     0.5443      1.016        139        640: 100%|██████████| 9/9 [00:01<00:00,  6.41it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.02it/s]


                   all         30        135      0.787      0.891      0.915      0.667

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    105/500       4.5G     0.8127     0.5761      1.041        118        640: 100%|██████████| 9/9 [00:01<00:00,  6.49it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.01it/s]

                   all         30        135      0.881      0.855      0.925      0.671



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    106/500      4.49G     0.7901     0.5545      1.019        114        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.00it/s]

                   all         30        135      0.915      0.855      0.946      0.694



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    107/500      4.57G     0.8056     0.5767      1.027        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.56it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.91it/s]

                   all         30        135      0.913      0.864      0.953      0.708



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    108/500       4.5G     0.8033      0.559      1.033        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.48it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135        0.8      0.871      0.926      0.678



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    109/500      4.52G     0.7822       0.56      1.021        146        640: 100%|██████████| 9/9 [00:01<00:00,  6.55it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135      0.857      0.831       0.91      0.661



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    110/500       4.5G     0.7541     0.5191      1.004        133        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.92it/s]

                   all         30        135      0.816      0.771       0.87      0.624



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    111/500      4.51G     0.7526     0.5107      1.007        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.46it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135       0.87      0.798       0.88      0.579



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    112/500      4.49G     0.7636     0.5282      1.017        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.31it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.08it/s]


                   all         30        135      0.751       0.77      0.835      0.555

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    113/500      4.52G     0.7883     0.5448      1.018        153        640: 100%|██████████| 9/9 [00:01<00:00,  6.56it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.87it/s]

                   all         30        135       0.86      0.851       0.89      0.608



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    114/500      4.51G     0.7778     0.5408      1.013        132        640: 100%|██████████| 9/9 [00:01<00:00,  6.25it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.923      0.894       0.95      0.703



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    115/500      4.56G     0.7742     0.5303      1.032         95        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.91it/s]

                   all         30        135      0.808       0.87      0.937      0.718



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    116/500      4.48G     0.7614     0.5209      1.017        126        640: 100%|██████████| 9/9 [00:01<00:00,  6.33it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.81it/s]

                   all         30        135      0.876      0.887      0.956      0.726



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    117/500      4.52G      0.744     0.5124      0.999        120        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.92it/s]

                   all         30        135      0.898      0.922      0.964      0.726



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    118/500       4.5G     0.7432     0.5332      1.011        108        640: 100%|██████████| 9/9 [00:01<00:00,  6.42it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.78it/s]

                   all         30        135      0.797      0.814      0.895      0.678



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    119/500      4.49G     0.7572     0.5384       1.02        134        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135      0.835      0.735       0.86       0.63



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    120/500       4.5G     0.7494     0.5136      1.012         97        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.875      0.891      0.948      0.694



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    121/500      4.52G     0.7386     0.5051      1.002        128        640: 100%|██████████| 9/9 [00:01<00:00,  6.55it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.99it/s]

                   all         30        135      0.888      0.849       0.92      0.678



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    122/500      4.48G     0.7275     0.5191      1.009         99        640: 100%|██████████| 9/9 [00:01<00:00,  6.27it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.927      0.885      0.948      0.719



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    123/500      4.54G       0.76     0.5458      1.022        113        640: 100%|██████████| 9/9 [00:01<00:00,  6.56it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.81it/s]

                   all         30        135      0.932      0.889      0.951      0.726



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    124/500      4.51G     0.7478     0.5385      1.009        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.85it/s]

                   all         30        135      0.937      0.855      0.935      0.705



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    125/500       4.5G     0.7285     0.5109     0.9989        120        640: 100%|██████████| 9/9 [00:01<00:00,  6.54it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.84it/s]

                   all         30        135      0.873      0.802      0.904      0.669



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    126/500      4.47G     0.7484     0.5269     0.9997        135        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.87it/s]

                   all         30        135      0.891      0.889      0.953      0.717



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    127/500       4.5G      0.759     0.5208     0.9962        112        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135      0.948      0.935      0.969        0.7



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    128/500      4.51G     0.7837     0.5428      1.019         80        640: 100%|██████████| 9/9 [00:01<00:00,  6.28it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135      0.925      0.913      0.957       0.71



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    129/500       4.5G     0.7639     0.5165      1.004        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.54it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.83it/s]

                   all         30        135      0.938      0.924      0.965      0.712



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    130/500      4.53G     0.7109      0.503     0.9931        134        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.98it/s]

                   all         30        135      0.918      0.884      0.952       0.69



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    131/500      4.52G     0.7061     0.4864     0.9857        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.927      0.863      0.945      0.705



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    132/500      4.54G     0.7019     0.4902     0.9859        135        640: 100%|██████████| 9/9 [00:01<00:00,  6.40it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.889      0.895      0.924      0.715



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    133/500      4.51G     0.7249     0.5082     0.9936        137        640: 100%|██████████| 9/9 [00:01<00:00,  6.52it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135      0.909      0.915      0.931      0.709



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    134/500      4.49G     0.7014      0.493     0.9867        145        640: 100%|██████████| 9/9 [00:01<00:00,  6.35it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.83it/s]

                   all         30        135      0.935      0.943      0.965      0.718



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    135/500      4.55G     0.6954     0.5002     0.9907        118        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.888      0.965       0.95      0.722



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    136/500      4.51G     0.7234     0.5022     0.9905        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135      0.883       0.84      0.922      0.694



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    137/500      4.51G     0.7108     0.5036     0.9944        109        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  5.00it/s]

                   all         30        135      0.892      0.931       0.95      0.739



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    138/500      4.49G     0.7363     0.5107      1.008        111        640: 100%|██████████| 9/9 [00:01<00:00,  6.38it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.944      0.921      0.967      0.764



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    139/500      4.52G     0.7022     0.4819     0.9826        128        640: 100%|██████████| 9/9 [00:01<00:00,  6.60it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.923      0.902      0.965      0.731



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    140/500      4.48G     0.7168     0.5133      1.003        128        640: 100%|██████████| 9/9 [00:01<00:00,  6.39it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135      0.877      0.882      0.943      0.692



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    141/500      4.53G     0.7161     0.5061     0.9956        106        640: 100%|██████████| 9/9 [00:01<00:00,  6.57it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.905      0.849      0.949       0.71



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    142/500      4.48G     0.6989     0.4762     0.9837        142        640: 100%|██████████| 9/9 [00:01<00:00,  6.29it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.74it/s]

                   all         30        135      0.916        0.9      0.963      0.742



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    143/500      4.53G     0.7133     0.5074     0.9835        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.50it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.91it/s]

                   all         30        135      0.956       0.89      0.951       0.74



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    144/500      4.48G     0.6931     0.4882     0.9875        129        640: 100%|██████████| 9/9 [00:01<00:00,  6.33it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135      0.934      0.882       0.95      0.732



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    145/500      4.52G     0.6973     0.4872     0.9813        112        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.97it/s]

                   all         30        135      0.944      0.907      0.957      0.715



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    146/500      4.56G     0.7097     0.4934     0.9915        127        640: 100%|██████████| 9/9 [00:01<00:00,  6.32it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135      0.966      0.947      0.972      0.734



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    147/500      4.51G     0.7003     0.4811     0.9956        100        640: 100%|██████████| 9/9 [00:01<00:00,  6.45it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.91it/s]

                   all         30        135      0.962      0.889       0.96      0.737



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    148/500      4.51G     0.6757      0.466     0.9783        143        640: 100%|██████████| 9/9 [00:01<00:00,  6.39it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.86it/s]

                   all         30        135       0.98      0.898      0.965      0.759



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    149/500      4.44G     0.6929     0.4697     0.9881        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.52it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.99it/s]

                   all         30        135      0.958      0.896      0.957      0.735



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    150/500      4.51G     0.6926     0.4722     0.9816        133        640: 100%|██████████| 9/9 [00:01<00:00,  6.30it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.84it/s]

                   all         30        135      0.935      0.872      0.943      0.707



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    151/500      4.49G     0.7076      0.482     0.9924        145        640: 100%|██████████| 9/9 [00:01<00:00,  6.43it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.88it/s]

                   all         30        135      0.955      0.857      0.959      0.733



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    152/500      4.57G     0.7032      0.486     0.9842        132        640: 100%|██████████| 9/9 [00:01<00:00,  6.29it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135      0.932      0.923      0.958      0.737



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    153/500      4.55G     0.6806     0.4623      0.983        116        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135       0.95      0.879      0.955      0.729



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    154/500      4.47G     0.6592     0.4646     0.9814        112        640: 100%|██████████| 9/9 [00:01<00:00,  6.31it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.87it/s]

                   all         30        135      0.996      0.914      0.968      0.761



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    155/500      4.53G     0.6535     0.4518     0.9741        145        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.79it/s]

                   all         30        135      0.907      0.921      0.958       0.71



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    156/500      4.48G     0.6663     0.4601     0.9728        154        640: 100%|██████████| 9/9 [00:01<00:00,  6.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.95it/s]

                   all         30        135      0.937      0.838      0.954       0.72



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    157/500      4.52G     0.6666     0.4558     0.9716        135        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.919      0.881      0.943      0.715



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    158/500      4.45G     0.6775     0.4666      0.982        119        640: 100%|██████████| 9/9 [00:01<00:00,  6.37it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.951      0.882      0.963      0.743



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    159/500      4.52G      0.688     0.4687     0.9729        163        640: 100%|██████████| 9/9 [00:01<00:00,  6.52it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.96it/s]

                   all         30        135      0.902      0.961      0.961      0.744



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    160/500      4.54G     0.6469     0.4532     0.9639        102        640: 100%|██████████| 9/9 [00:01<00:00,  6.26it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.87it/s]

                   all         30        135      0.856      0.888      0.938      0.716



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    161/500      4.54G       0.68     0.4544     0.9771        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.53it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.847      0.911      0.942      0.733



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    162/500      4.49G     0.6645     0.4663     0.9808        110        640: 100%|██████████| 9/9 [00:01<00:00,  6.34it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.85it/s]

                   all         30        135      0.928      0.935      0.972      0.761



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    163/500       4.5G     0.6411     0.4374     0.9756        118        640: 100%|██████████| 9/9 [00:01<00:00,  6.51it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.92it/s]

                   all         30        135      0.939      0.884      0.962       0.76



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    164/500      4.48G     0.6542     0.4466     0.9621        104        640: 100%|██████████| 9/9 [00:01<00:00,  6.40it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.92it/s]

                   all         30        135      0.959      0.845      0.945       0.74



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    165/500       4.5G     0.6803       0.45      0.972        138        640: 100%|██████████| 9/9 [00:01<00:00,  6.44it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.94it/s]

                   all         30        135      0.914       0.91      0.942      0.725



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    166/500      4.51G     0.6655     0.4447     0.9566        125        640: 100%|██████████| 9/9 [00:01<00:00,  6.10it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.80it/s]

                   all         30        135      0.946      0.918      0.963      0.761



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    167/500      4.54G     0.6345     0.4363     0.9593        106        640: 100%|██████████| 9/9 [00:01<00:00,  6.45it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.93it/s]

                   all         30        135      0.961      0.906      0.964      0.759



      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size


    168/500      4.48G     0.6657     0.4522     0.9796        122        640: 100%|██████████| 9/9 [00:01<00:00,  6.32it/s]
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  4.90it/s]

                   all         30        135      0.909      0.955      0.967       0.76
EarlyStopping: Training stopped early as no improvement observed in last 30 epochs. Best results observed at epoch 138, best model saved as best.pt.
To update EarlyStopping(patience=30) pass a new patience value, i.e. `patience=300` or use `patience=0` to disable EarlyStopping.



168 epochs completed in 0.120 hours.
Optimizer stripped from runs/detect/train15/weights/last.pt, 6.3MB
Optimizer stripped from runs/detect/train15/weights/best.pt, 6.3MB

Validating runs/detect/train15/weights/best.pt...
Ultralytics 8.3.49 🚀 Python-3.9.16 torch-2.0.1+cu117 CUDA:3 (NVIDIA RTX A5000, 24123MiB)
YOLOv8n summary (fused): 168 layers, 3,006,233 parameters, 0 gradients, 8.1 GFLOPs


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95): 100%|██████████| 1/1 [00:00<00:00,  2.66it/s]


                   all         30        135      0.944      0.922      0.966      0.764
                   bsf         10         46      0.923      0.913      0.962      0.707
                  crpf          9         39      0.948      0.897      0.981      0.776
             jk_police         11         50       0.96      0.955      0.957      0.808
Speed: 0.1ms preprocess, 0.7ms inference, 0.0ms loss, 1.2ms postprocess per image
Results saved to runs/detect/train15


## Training Results

**168 epochs completed** in **0.120 hours**.

#### Model Weights:
- Optimizer stripped from `runs/detect/train15/weights/last.pt`, **6.3MB**
- Optimizer stripped from `runs/detect/train15/weights/best.pt`, **6.3MB**

---

### Validation Results

**Validating** `runs/detect/train15/weights/best.pt`...

```plaintext
Ultralytics 8.3.49 🚀 Python-3.9.16 torch-2.0.1+cu117 CUDA:3 (NVIDIA RTX A5000, 24123MiB)
YOLOv8n summary (fused): 168 layers, 3,006,233 parameters, 0 gradients, 8.1 GFLOPs

Class	Images	Instances	Box(P)	R	mAP50	mAP50-95
All	30	135	0.944	0.922	0.966	0.764
BSF	10	46	0.923	0.913	0.962	0.707
CRPF	9	39	0.948	0.897	0.981	0.776
JK Police	11	50	0.960	0.955	0.957	0.808
